structs/blocks.go

```diff
@@ -7,8 +7,11 @@ import (
 )
 
 type SubmitBlockRequest interface {
+	BidTrace() types.BidTrace
```

Comment:
I was trying hard not to do that, and limit this to a per usage getter, can we create TraceBlockHash and TraceParentHash by any chance ?:) Or create those validates inside `SubmitBlockRequest` ? Like the comparison function that would do the checks internally ? :)

Reply:
Sure